packages/lesswrong/lib/sql/Table.ts

```diff
@@ -20,6 +20,7 @@ import { forumTypeSetting, ForumTypeString } from "../instanceSettings";
 class Table<T extends DbObject> {
   private fields: Record<string, Type> = {};
   private indexes: TableIndex<T>[] = [];
+  wal = true;
```

Comment:
_probably_ this should be `private` with a corresponding `isLogged()` accessor, since I don't think we ever want to change mutate this field on a `Table` instance

Reply:
reasonable